app/src/main/java/net/squanchy/notification/NotificationCreator.kt

```diff
@@ -116,7 +116,7 @@ class NotificationCreator(private val context: Context) {
         val eventDetailIntent = EventDetailsActivity.createIntent(context, eventId)
         taskBuilder.addNextIntent(eventDetailIntent)
 
-        return taskBuilder.getPendingIntent(0, PendingIntent.FLAG_CANCEL_CURRENT)
+        return taskBuilder.getPendingIntent(0, PendingIntent.FLAG_CANCEL_CURRENT)!!
```

Comment:
New `@Nullable` annotations in the SDK, but the return value can only be `null` if you pass `FLAG_NO_CREATE` — which we don't.